neo4j/v4.x.x/Dockerfile

```diff
@@ -1,6 +1,9 @@
 # Official Neo4j Docker Images available at https://hub.docker.com/_/neo4j
 FROM neo4j:4.2.2
 
+# OPTIONAL: For arm64 (M1 Macs) comment the previous line, and uncomment the next line.
```

Comment:
This is such a great addition! I'm getting this project setup on my new 14" MacBook before reviewing your PR, and will use this line as well.

Reply:
Happy to have helped. Also, I've found that most other docker imags don't use a different repo url for arm64 compatible builds.
Enjoy your new MacBook!

Reply:
Yeah! I'm also seeing in the live announcement from Neo4j - happening right now at https://www.youtube.com/embed/19m4MBYBnCs?autoplay=1&feature=oembed&wmode=opaque to discuss the new 4.4 release - that the generic base Neo4j image should work on arm64 as of tomorrow. That's exciting! We'll have to see how that goes.

Reply:
Also I am happy to see my workload is lightening up a little, and I can finally take a look at your PR by the end of the weekend. Hopefully sooner!

Reply:
Cheers bud :) Look forward to seeing what you think...
BTW I coded up the neo4j adapter, so you might be the about the 2nd person to spin it up and test it out 😄

Reply:
Hey hey! Just coming up for air after being buried under a mountain of things. I didn't want you to think I forgot about you or this PR. It's definitely on my radar - albeit a couple days out at this point given the mountain of stuff I'm digging out from 🏔

Reply:
Hey! No sweat - seriously. I get how workloads can be huge, so please don't feel commited to take a look at this quickly. The last thing I want is to know I'm contributing towards stresses. Projects like this should be fun, so just take a look if and when you feel like! ✌️